src/Proto3/Wire/Encode.hs

```diff
@@ -122,6 +129,13 @@ instance Show MessageBuilder where
     where
       bytes' = toLazyByteString builder
 
+etaMessageBuilder :: (a -> MessageBuilder) -> (a -> MessageBuilder)
+etaMessageBuilder = (MessageBuilder .) . RB.etaBuildR . (unMessageBuilder .)
```

Comment:
I think you can just do:

```haskell
etaMessageBuilder = coerce RB.etaBuildR
```

Reply:
Sure; I think that is equivalent.

Reply:
5fefcb2713f0e08d0d170cfa09397b314bce16f9 changes this to use `coerce`.